text/0000-href-to.md

```diff
@@ -0,0 +1,47 @@
+- Start Date: 2015-09-12
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Summary
+
+This RFC proposes the addition of a `href-to` helper which is a lightweight alternative to the `link-to` helper.
+
+# Motivation
+
+For performance sensitive applications like [Intercom](https://www.intercom.io/), `link-to` can be quite heavy. We've found many cases where all we want is to link to a route. `link-to` creates a component with a set of attribute and class bindings which we don't need. In contrast, the `href-to` helper would simply render a URL at render time.
```

Comment:
We should definitely make link-to faster, but the two ideas are not opposed to each other. For example, one way to make it faster is to have it take advantage of template helpers like this one.